assets/LICENSE-APACHE

```diff
@@ -0,0 +1,201 @@
+                            Apache License
```

Comment:
I'm not sure; AFAIK licenses should be distributed with the source code. Now, a cargo package contains the source code, but not the top-level license. That's the reason I put it in there, just to not cause any trouble. But I personally don't care whether or not there's a license in these crates.